Fails on macos 10.15:

```
builder for '/nix/store/6r2dz17p6ahmkaaavl5mg2ygxksk85m8-python3.9-aiofiles-0.7.0.drv' failed with exit code 1; last 10 log lines:
              return
      
          try:
  >           result = self.fn(*self.args, **self.kwargs)
  E           OSError: [Errno 38] Socket operation on non-socket
  
  /nix/store/ha9svlp7cbvgibrz773nfrjn6phgfqyw-python3-3.9.6/lib/python3.9/concurrent/futures/thread.py:52: OSError
  =========================== short test summary info ============================
  FAILED tests/test_os.py::test_sendfile_file - OSError: [Errno 38] Socket oper...
  ======================== 1 failed, 167 passed in 4.47s =========================
```

--- a/pkgs/development/python-modules/aiofiles/default.nix
+++ b/pkgs/development/python-modules/aiofiles/default.nix
@@ -1,4 +1,5 @@
-{ lib
+{ stdenv
+, lib
 , buildPythonPackage
 , fetchFromGitHub
 , poetry-core
@@ -28,6 +29,10 @@ buildPythonPackage rec {
   checkInputs = [
     pytest-asyncio
     pytestCheckHook
+  ];
+
+  disabledTests = lib.optionals stdenv.isDarwin [
+    "test_sendfile_file"
   ];
 
   pythonImportsCheck = [ "aiofiles" ];

